cms/templates/js/timed-examination-preference-editor.underscore

```diff
@@ -1,50 +1,65 @@
 <form>
-    <h3 class="modal-section-title"><%- gettext('Timed Exam') %></h3>
+    <h3 class="modal-section-title"><%- gettext('Additional Options:') %></h3>
+
     <div class="modal-section-content has-actions">
+
         <div class='exam-time-list-fields'>
             <ul class="list-fields list-input">
-                <li class="field field-checkbox checkbox-cosmetic">
-                    <input type="checkbox" id="id_timed_examination" name="timed_examination" class="input input-checkbox" />
-                    <label for="id_timed_examination" class="label">
-                      <i class="icon fa fa-check-square-o input-checkbox-checked"></i>
-                      <i class="icon fa fa-square-o input-checkbox-unchecked"></i>
-                      <%- gettext('This exam is timed') %>
-                    </label>
+                <li class="field-radio">
+                    <input type="radio" id="id_not_timed" name="proctored" class="input input-radio" checked="checked"/>
+                    <label for="id_not_timed" class="label">
+                      <%- gettext('None') %>
+                    </label>  
                 </li>
             </ul>
         </div>
+
         <div class='exam-time-list-fields'>
+            <ul class="list-fields list-input">
+                <li class="field-radio">
+                    <input type="radio" id="id_timed_exam" name="proctored" class="input input-radio" />
+                    <label for="id_timed_exam" class="label">
+                      <%- gettext('Timed') %>
+                    </label>
+                </li>
+                <p class='field-message'> <%- gettext('Use a timed exam to limit the time learners can spend on problems in this subsection. Learners must submit answers before the time expires. You can allow additional time on per learner basis through the Instructor Dashboard.') %> </p>
```

Comment:
Add a unique `id` here, then add `aria-describedby="[said id]"` to the `input` it's associated with above.